app/assets/stylesheets/application_letters.css

```diff
@@ -31,10 +31,9 @@ ul#eating-habit--list > li {
 	margin-left: 3px;
 }
 
-.application-details-table.table {
-  table-layout: fixed;
+dl:not(.dl-horizontal) dd {
```

Comment:
I know it is nitpicky, but indentions and less :) Decide on your own, if you want to fix it.